sources/Application/Model/BuildNumber.h

```diff
@@ -1 +1 @@
-#define BUILD_COUNT "003"
+#define BUILD_COUNT "001"
```

Comment:
there is no need to change this as this is just a placeholder value that is replaced in GitHub CI (Actions)